tests/e2e/integ-suite-kind.sh

```diff
@@ -52,16 +62,67 @@ function setup_kind_registry() {
   fi
 
   # https://docs.tilt.dev/choosing_clusters.html#discovering-the-registry
-  # TODO get context/config from existing variables
-  kind export kubeconfig --name="${KIND_CLUSTER_NAME}"
-  for node in $(kind get nodes --name="${KIND_CLUSTER_NAME}"); do
-    kubectl annotate node "${node}" "kind.x-k8s.io/registry=localhost:${KIND_REGISTRY_PORT}" --overwrite;
+  for cluster in $(kind get clusters); do
+    # TODO get context/config from existing variables
+    # Avoid adding the registry to excluded clusters. Use when you have multiple clusters running.
+    if [[ "${KIND_EXCLUDE_CLUSTERS}" == *"${cluster}"* ]]; then
+      continue
+    fi
+
+    kind export kubeconfig --name="${cluster}"
+    for node in $(kind get nodes --name="${cluster}"); do
+      kubectl annotate node "${node}" "kind.x-k8s.io/registry=localhost:${KIND_REGISTRY_PORT}" --overwrite;
+    done
   done
 }
 
-KUBECONFIG="${ARTIFACTS}/config" setup_kind_cluster "${KIND_CLUSTER_NAME}" "" "" "true" "true"
-setup_kind_registry
+# Create a self signed root CA and intermediate CAs and push them to each cluster.
+function setup_ca() {
```

Comment:
I'm wondering if we want to do this here at all or if it should be done inside `pushIntermediateCA` in the test? Then the tests aren't reliant on connectivity to `https://raw.githubusercontent.com` or the makefile changing.

Reply:
Moved